src/Executables/ExportCoordinates/ExportCoordinates.hpp

```diff
@@ -205,8 +205,7 @@ struct Metavariables {
       : tt::ConformsTo<Options::protocols::FactoryCreation> {
     using factory_classes = tmpl::map<
         tmpl::pair<Event, tmpl::list<Events::Completion>>,
-        tmpl::pair<Trigger, tmpl::append<Triggers::logical_triggers,
-                                         Triggers::time_triggers>>>;
+        tmpl::pair<Trigger, tmpl::list<Triggers::TimeCompares>>>;
```

Comment:
Why delete the other triggers? They could potentially be useful to observe coordinates at exactly those times that another simulation observed things at.

Reply:
The triggers don't control the observations, they're just used to terminate the "evolution".  The observations happen after every time step.

The only other trigger that I could see being useful is `SlabCompares`.  I'll add that one back in.

Reply:
Squashed directly.  Trivial diff:
```patch
diff --git a/src/Executables/ExportCoordinates/ExportCoordinates.hpp b/src/Executables/ExportCoordinates/ExportCoordinates.hpp
index d5c45ab81..241676053 100644
--- a/src/Executables/ExportCoordinates/ExportCoordinates.hpp
+++ b/src/Executables/ExportCoordinates/ExportCoordinates.hpp
@@ -48,6 +48,7 @@
 #include "ParallelAlgorithms/Initialization/Actions/RemoveOptionsAndTerminatePhase.hpp"
 #include "Time/Actions/AdvanceTime.hpp"
 #include "Time/TimeSteppers/TimeStepper.hpp"
+#include "Time/Triggers/SlabCompares.hpp"
 #include "Time/Triggers/TimeCompares.hpp"
 #include "Utilities/Blas.hpp"
 #include "Utilities/ErrorHandling/FloatingPointExceptions.hpp"
@@ -205,7 +206,8 @@ struct Metavariables {
       : tt::ConformsTo<Options::protocols::FactoryCreation> {
     using factory_classes = tmpl::map<
         tmpl::pair<Event, tmpl::list<Events::Completion>>,
-        tmpl::pair<Trigger, tmpl::list<Triggers::TimeCompares>>>;
+        tmpl::pair<Trigger, tmpl::list<Triggers::SlabCompares,
+                                       Triggers::TimeCompares>>>;
   };
 
   enum class Phase { Initialization, RegisterWithObserver, Export, Exit };
```

Answering another possible interpretation of the original question: I had to either remove Triggers from this list or add the TimeSequence factory information, and this change made more sense to me.